db/migrate/20230821233937_add_status_to_user_mentee_applications.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToUserMenteeApplications < ActiveRecord::Migration[7.0]
+  def change
+    add_column :user_mentee_applications, :status, :integer
+  end
+end
```

Comment:
We don't need this. There shouldn't be a `status` on the application directly.

Reply:
oh yeah 😅 forgot to delete this migration